src/panels/config/automation/manual-automation-editor.ts

```diff
@@ -642,6 +670,41 @@ export class HaManualAutomationEditor extends LitElement {
     }
   }
 
+  private _resizeSidebarWidth = () => {
+    this.style.setProperty(
+      "--sidebar-dynamic-width",
+      `${this._sidebarWidthPx}px`
+    );
+  };
+
+  private _resizeSidebar(ev) {
+    ev.stopPropagation();
+    const delta = ev.detail.deltaInPx as number;
+
+    // set initial resize width to add / reduce delta from it
+    if (!this._prevSidebarWidthPx) {
+      this._prevSidebarWidthPx =
+        this._sidebarElement?.clientWidth || SIDEBAR_DEFAULT_WIDTH;
+    }
+
+    const widthPx = delta + this._prevSidebarWidthPx;
+
+    this._sidebarWidthPx = Math.min(
+      Math.max(widthPx, SIDEBAR_MIN_WIDTH_DOCKED_NAVBAR),
+      SIDEBAR_MAX_WIDTH
+    );
```

Comment:
I dont think we need to calculate anything here? The css will handle it.

Reply:
Yes the css handles it. I keep this in there to avoid strange values in the storage like -682. 

But if you think its too expensive lets remove it.